coding-concepts.js

```diff
@@ -16,44 +16,49 @@ const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2022"
-// console.log(cohort.length)
+console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: it will come back as number 10
+// b) Verify and explain: the .lenght counts the number of characters inside the string, learn is 5 and 2022 is 4 but the space in between those counts as a character so in total is 10.
 
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
-// console.log(greeting[4])
+console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: I belive it will show one of the letters, the letter "o" of Hello
+// b) Verify and explain: I was correct, it returned the letter "o" I know that in coding when we search for the placing of a letter it starts with 0, so if we start counting from zero in the "Hello world" the letter o is the fourth.
 
 
 // --------------------3) What will this log?
 
 const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
-// console.log(languages[index])
+console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: I think that it will show the either the first letter of the first word or the first word "JavaScript" but the index 1 is throwing me off a little
+// b) Verify and explain: It showed "Ruby", because when the index was added it was looking for the first value (Ruby) so because the string is zeroth indexed the 0 is "JavaScript"
 
 
 // --------------------4) What will this log?
 
-const weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
-
-// a) Your answer:
-// b) Verify and explain:
+ cosnt weekendDays = ["saturday", "sunday"]
+console.log(weekendDays.toUpperCase())
 
+// a) Your answer: the value of weekendDays will come back in all upper case letters.
+// b) Verify and explain: I get an error saying that .toUpperCase is not a function. I know .toUpperCase upper cases the letters because I saw it in the intro and also on the syllabus. Maybe it has something to do with the const but I doubt it
```

Comment:
`.toUpperCase()` is a string built-in method. the variable `weekendDays` is holding an array. The first thing we have to look at what datatype we have then apply the appropriate built-in methods for that datatype.